include/internal/libfreenect2/depth_packet_processor.h

```diff
@@ -175,6 +175,25 @@ class OpenCLDepthPacketProcessor : public DepthPacketProcessor
 private:
   OpenCLDepthPacketProcessorImpl *impl_;
 };
+
 #endif // LIBFREENECT2_WITH_OPENCL_SUPPORT
+
+class DumpDepthPacketProcessor : public DepthPacketProcessor
```

Comment:
Can you explain why you want to dump depth packets? The raw depth packets will require P0, X, and Z tables to decode. The tables are pulled from the Kinect at run time. So far my impression is that the raw depth packets are too big to be practical (85MB/s).